src/scripts/alleles/G_domain.py

```diff
@@ -0,0 +1,185 @@
+from Bio import SeqIO, BiopythonWarning, BiopythonParserWarning
+import csv
+import sys
+import os
+sys.path.append(os.getcwd())
+from io import StringIO
+import warnings
+import logging
+
+EXCLUDED_GENES = {
+    "BTN3A",
+    "CD1",
+    "DOA",
+    "DOB",
+    "V",
+    "P",
+    "S",
+    "Y",
+    "MICA",
+    "MICB",
+    "MR1",
+    "HFE",
+    "J",
+    "L",
+    "TAP1",
+    "DMB",
+    "DMA",
+    "T",
+    "V",
+    "W",
+    "K",
+    "TAP2",
+    "U",
+    "H",
+    "DRB8",
+    "DRB9",
+    "DRB7",
+    "DRB6",
+    "DPB2",
+    "DPA2",
+    "DQA2",
+    "DRB2"
+}
+
+logging.basicConfig(filename = 'biopython.log', filemode = 'w', level = logging.DEBUG )
```

Comment:
This log file should probably go in the `build` directory. This will log everything from both Biopython and this script, so `biopython.log` is not the most informative name.

Is there a reason you're using a log file instead of logging to the console? Looking at the log file, there is a LOT of stuff and it's hard to tell what is relevant. For example, I see a lot of:
```
WARNING:py.warnings:/Users/jackson/opt/anaconda3/lib/python3.8/site-packages/Bio/GenBank/Scanner.py:681: BiopythonParserWarning: EMBL sequence line missing coordinates
  warnings.warn(
```

There's also almost 50,000 lines in the log file.

Reply:
Sorry, I should have mentioned this before.  This warning, when I am parsing the build/hla.dat file specifically happens when there is no nucleotide sequence like [here](https://www.ebi.ac.uk/Tools/dbfetch/dbfetch?db=imgthla;id=HLA06639). I spent some time trying to find a way around this, but I couldn't because the problem is with how Biopython parses the file.  The rest of the lines are just there to help in debugging (specifically which entry in build/hla.dat is causing problems). I am pushing an update to fix this.